adb/adb_commands.py

```diff
@@ -133,10 +133,11 @@ def Install(self, apk_path, destination_dir=None, timeout_ms=None):
                       timeout_ms=timeout_ms)
 
   def Push(self, source_file, device_filename, mtime='0', timeout_ms=None):
-    """Push source_file to file on device.
+    """Push a file or directory to the device.
```

Comment:
Does this actually work with directories? I thought that PR was still in progress?